images/all-in-one/Dockerfile

```diff
@@ -89,16 +90,10 @@ RUN set -eux; \
         'amd64') \
             url="https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz"; \
             ibm_cloud_url="https://download.clis.cloud.ibm.com/ibm-cloud-cli/${IBM_CLOUD_VERSION}/IBM_Cloud_CLI_${IBM_CLOUD_VERSION}_amd64.tar.gz"; \
-            curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg; \
-            mv bazel.gpg /etc/apt/trusted.gpg.d/; \
-            echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list; \
-            apt update && apt install -y bazel=$BAZEL_VERSION; \
             ;; \
         'ppc64el') \
             url="https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-ppc64le.tar.gz"; \
             ibm_cloud_url="https://download.clis.cloud.ibm.com/ibm-cloud-cli/${IBM_CLOUD_VERSION}/IBM_Cloud_CLI_${IBM_CLOUD_VERSION}_ppc64le.tar.gz"; \
-            wget -O /usr/local/bin/bazel "https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel/ubuntu_18.04/bazel_bin_ppc64le_$BAZEL_VERSION" --progress=dot:giga; \
-            chmod +x /usr/local/bin/bazel; \
             ;; \
         's390x') \
```

Comment:
As mentioned [here](https://github.ibm.com/powercloud/container-dev/issues/1573#issuecomment-43603570), seems we may remove the s390x-related code

Reply:
yes, lemme remove the whole block itself.